shardingsphere-infra/shardingsphere-infra-util/src/main/java/org/apache/shardingsphere/infra/util/exception/sql/ShardingSphereSQLException.java

```diff
@@ -40,6 +40,11 @@ public ShardingSphereSQLException(final SQLState sqlState, final int vendorCode,
     }
     
     public ShardingSphereSQLException(final String sqlState, final int vendorCode, final String reason, final Object... messageArguments) {
+        this(null, sqlState, vendorCode, reason, messageArguments);
+    }
+
+    public ShardingSphereSQLException(final Exception cause, final String sqlState, final int vendorCode, final String reason, final Object... messageArguments) {
+        super(cause);
```

Comment:
It's better to set Exception as the last parameter.

Reply:
There can't be more pararmeter after indefinite length array